### PR TITLE
fix(InvestigatorsTable): refactor investigator list fetch

### PR DESCRIPTION
**Pull request by doronlnd84**
Fixing bug [1305](https://dev.azure.com/spectrumFactory/CoronaI/_sprints/backlog/CoronaI%20Team/CoronaI/Sprint%2013?workitem=1305)
Due to performance issue in Prod environment. I change investigator fetch - now fetching by sql query called from function (instead of graphQL query). Thus, I was forced to fit  some client parsing to the returning object

### CODE REVIEW

File: server/src/ClientToDBAPI/UsersRoute/mainRoute.ts [hunk 1]
```diff
@@ -226,15 +226,17 @@ usersRoute.get('/group', adminMiddleWare, (request: Request, response: Response)
...
-    const parameters = { investigationGroup: +response.locals.user.investigationGroup };
+    const parameters = { inputCountyId: +response.locals.user.investigationGroup };
     groupLogger.info(launchingDBRequestLog(parameters), Severity.LOW);
 
     graphqlRequest(GET_ACTIVE_GROUP_USERS, response.locals, parameters)
         .then(result => {
             groupLogger.info(validDBResponseLog, Severity.LOW);
-            const users: User[] = result.data.allUsers.nodes.map((user: any) => ({
+            const resData = JSON.parse(result.data.getInvestigatorListByCountyFunction.json);
+            resData.map((user: any) => (console.log('languages',user.languages)));
```
Comment (eladyashar) [inline server/src/ClientToDBAPI/UsersRoute/mainRoute.ts:236]: remove debug logs

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const InvestigatorsTable: React.FC<Props> = ({ investigators, selectedRow, setSe
                                 {
                                     Object.keys(TableHeaders).map((cellHeader: string) => (
                                         <TableCell key={cellHeader}>
-                                            { getTableCell(investigator, cellHeader) }
+                                            { getTableCell(investigator, cellHeader.toLocaleLowerCase()) }
```
Comment (eladyashar) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx:70]: what the reason for the casting ?

**Review by eladyashar — APPROVED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx (hunk 2 above)
Comment (doronlnd84) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx:70]: the json object returned from DB has lowercase property name.  I was forced to do that.

File: server/src/ClientToDBAPI/UsersRoute/mainRoute.ts (hunk 1 above)
Comment (doronlnd84) [reply server/src/ClientToDBAPI/UsersRoute/mainRoute.ts:236]: will be fixed

**Review by eladyashar — APPROVED** [review]

**Review by OrelKaravani — APPROVED** [review]

**Pull request merged**